plugins/woocommerce-admin/bin/package-update-textdomain.js

```diff
@@ -0,0 +1,31 @@
+const wpTextdomain = require( 'wp-textdomain' );
+
+const dirs = process.argv.slice( 2 );
+
+for ( dir of dirs ) {
+	let path = dir + '/**/*.php';
+	console.log("Processing: " + dir);
+	wpTextdomain( path, {
+		domain: 'woocommerce',
+		fix: true,
+		missingDomain: true,
+		variableDomain: true,
+		keywords: [
+			'__:1,2d',
+			'_e:1,2d',
+			'_x:1,2c,3d',
+			'esc_html__:1,2d',
+			'esc_html_e:1,2d',
+			'esc_html_x:1,2c,3d',
+			'esc_attr__:1,2d',
+			'esc_attr_e:1,2d',
+			'esc_attr_x:1,2c,3d',
+			'_ex:1,2c,3d',
+			'_n:1,2,4d',
+			'_nx:1,2,4c,5d',
+			'_n_noop:1,2,3d',
+			'_nx_noop:1,2,3c,4d',
+			'wp_set_script_translations:1,2d,3',
+		],
+	} );
+}
```

Comment:
I'll be making another PR to remove and publish these scripts somewhere, not in the tools folder as @ObliviousHarmony mentioned the tools folder probably isn't the best place for it to live out the rest of its life

But I think it's good to have the code inside this PR so that in the future if someone is doing commit history sleuthing he can find it right here in this PR